easybuild/easyconfigs/t/TreeShrink/TreeShrink-1.3.2-GCC-8.2.0-2.31.1-Python-3.7.2.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'PythonPackage'
+
+name = 'TreeShrink'
+version = '1.3.2'
```

Comment:
There is no official 1.3.2 version of this yet.
So please push the developers to release a 1.3.2 version first.

Reply:
@akesandgren it seems they are not creating the releases in github but they are labeling them with commits.
They have even labeled a 1.3.3.
https://github.com/uym2/TreeShrink/commits/master

Shall I push developers to create those releases in github?

Reply:
Yes, try to make them understand that they need to make an actual release, or a the very least a tag, although an actual release is much preferred. (A tag can be moved causing things to get out of sync)
Just having it named as v1.3.2 in a commit is not sufficient for people to be able to know what v1.3.2 really is.